docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,269 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the Home screen.
+
+To create a DALi widget application, you must:
+
+- Use the [Dali::WidgetApplication class](#widget-application-api).
+
+	> **Note**
+	> - The implementation of DALi widget application is similar to [DALi basic UI application](dali-basic-app.md), because the [Dali::WidgetApplication](../../api/mobile/latest/classDali_1_1WidgetApplication.html) class inherits from the [Dali::Application](../../api/mobile/latest/classDali_1_1Application.html) class.
+  > - Ensure you are familiar with the basic UI application details before tackling a widget application.
+
+- Ensure you start the event loop.
+
+- To implement your own widget class inherit the `Dali::Widget` class.
+
+![DALi widget application](./media/dali_widget_application.png)
+
+**Figure: DALi widget application**
+
+For more information on basics of creating a DALi widget application, see [Create a DALi Widget Application](#create).
+
+<a name="widget-application-api"></a>
```

Comment:
Hello, @neostom432.

In markdown, the sub title with "##" automatically generate link using the string.
For example as title is "Widget Application Class" here,
you can jump to here with \[Dali::WidgetApplication class\]\(\#**widget-application-class**\).

So, would you change the hyperlink in line to use this kind of links,  instead of \<a\> tag?

Reply:
@mijin85cho done